backup/backup.sh

```diff
@@ -46,9 +46,12 @@ xml_dump() {
     echo
     echo "XML backup"
     XML_DUMP_FILE=portal_xml_backup_${DATE_STRING}.gz
+    # Relevant namespaces: https://portal.mardi4nfdi.de/wiki/Special:NamespaceInfo
+    NAMESPACES=(0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 120 121 122 123 274 275 640 641 828 829 3000)
```

Comment:
Do we want to have the Wikibase items 120,122 in the XML output. this would be JSON wrapped into XML, I'm not sure if that's super useful. I think it would be better to export as TTL or JSON instead.

Reply:
I agree that it would be better to export the wikibase entities as JSON, but do you know of an easy way to do that? I couldn't find it here: https://www.mediawiki.org/wiki/Manual:Backing_up_a_wiki 
I think the important point is to ignore the namespaces that we currently use for Person:, Publication:, etc. I hope that that will be enough to at least fix the current problem.

Reply:
I think it will not be sufficient thus I'd go the other way around. If that succeeds we can include 120.

Reply:
We can use this https://github.com/wikimedia/mediawiki-extensions-Wikibase/blob/master/repo/maintenance/dumpJson.php to dump all the entities as a JSON file.

I am not sure how helpful that is for recovering purposes, though. Is there an easy way to load back all the entities to Wikibase from a JSON file?

Reply:
Ok, I'm gonna first merge this version to test how it goes. Later I can try saving the wikibase entities in a different json file.

Reply:
Sure, give it a try. I hope it completes in less than 24h.